source/jsonify.d.ts

```diff
@@ -6,6 +6,10 @@ import type {IsNever} from './is-never';
 import type {NegativeInfinity, PositiveInfinity} from './numeric';
 import type {TypedArray} from './typed-array';
 
+type HasAnyProperty<T extends object> = {
+    [K in keyof T]: IsAny<T[K]> extends true ? K : never;
```

Comment:
Whoops, sorry about that.

```suggestion
	[K in keyof T]: IsAny<T[K]> extends true ? K : never;
```